src/net.cpp

```diff
@@ -2856,7 +2856,7 @@ void CConnman::RecordBytesRecv(uint64_t bytes)
     nTotalBytesRecv += bytes;
 }
 
-void CConnman::RecordBytesSent(uint64_t bytes)
+void CConnman::RecordBytesSent(uint64_t bytes, bool increase_max_outbound)
```

Comment:
I don't find the name of this new variable particularly informative... how about `bypass_limit`?